cmd/cli/app/provider/provider_enroll.go

```diff
@@ -171,15 +170,15 @@ func enrollUsingOAuth2Flow(
 
 // callBackServer starts a server and handler to listen for the OAuth callback.
 // It will wait for either a success or failure response from the server.
-func callBackServer(ctx context.Context, cmd *cobra.Command, provider string, project string, port string,
-	done chan bool, client minderv1.OAuthServiceClient, openTime time.Time) {
+func callBackServer(ctx context.Context, cmd *cobra.Command, project string, port string, done chan bool,
```

Comment:
Why is port a `string` and not an `int`?  (I'm generally suspicious of multiple `string`-type arguments next to each other -- I've seen them get swapped more often than I'd like.

Reply:
No idea why it was like that. I changed it to be an int.